Cloud Application Manager/Deploying Anywhere/using-the-vmware-vcenter-private-datacenter.md

```diff
@@ -88,10 +88,14 @@ In order to deploy to a vCenter private datacenter, you must first provide infor
 2. Select **VMware vSphere**.
 
 3. Enter the endpoint URL for the vCenter server and a username, password to the vCenter API.
-   * **Note:** The endpoint URL must be in the form of **https://<servername>** or **https://<ipaddress>**. Be sure to use https and not http.
+   * **Note:** The endpoint URL can be a hostname or an IP address, and has to be prefixed with **https://**. Be sure to use https and not http.
 
    ![add-vsphere-provider-1.png](../../images/cloud-application-manager/add-vsphere-provider-1.png)
 
+Cloud Application Manager is switching to a more recent implementation of the vSphere API in order to provide long-term support for current and upcoming releases of vCenter products. By default, new providers use the newer implementation, while existing providers will stick to the legacy implementation.
```

Comment:
Hey Denes, thanks! These two sentences are great for next Release Notes! 😉 But, for KB article (it should not have temporal aspects such as `is switching` (KB articles are not time-based, while KB are), since if someone reads the article in 5 years time, we will not "be switching" then!) I would change them to explain the toggle first and its consequences without any temporal reference

Reply:
How about it now?